engine/statmetrics.go

```diff
@@ -240,8 +240,8 @@ type StatACD struct {
 // getValue returns acr.val
 func (acd *StatACD) getValue() time.Duration {
 	if acd.val == nil {
-		if (acd.MinItems > 0 && acd.Count < int64(acd.MinItems)) || (acd.Count == 0) {
-			acd.val = utils.DurationPointer(time.Duration((-1) * time.Nanosecond))
+		if acd.MinItems > 0 && acd.Count < int64(acd.MinItems) || acd.Count == 0 {
```

Comment:
We do not want to remove paranteses here either since it can change the grouping logic and affects the readability.